libs/application/templates/party-application/src/lib/PartyApplicationTemplate.ts

```diff
@@ -279,8 +306,9 @@ const PartyApplicationTemplate: ApplicationTemplate<
           ...context,
           application: {
             ...context.application,
-            // todo: get list of supreme court national ids
-            assignees: ['0000000000'],
+            assignees: assignees(
```

Comment:
Maybe rename the `assignees` function here to make it more clear this is a custom function maybe `getConstituencyAssignees`